includes/frontend/class-wcmp-checkout.php

```diff
@@ -350,7 +350,7 @@ public static function save_delivery_options($order_id)
             ]
             : null;
 
-        $deliveryOptions = empty($deliveryOptionsFromPost)
+        $deliveryOptions = ! array($deliveryOptionsFromPost)
```

Comment:
Waarom wordt er een array omheen gezet (en waarom wordt het zo geschreven)?

Reply:
De reden dat dit is gedaan is omdat de delivery_options (deliveryOptionsFromPost) niet meer leeg is, zodra er geen gebruikt wordt gemaakt van de delivery options. Er staat nu "null" (als string). Dit is dan ook de reden dat ik controleer of het een array is.

Reply:
Maar je controleert niet of het een array is, je stopt het in een array en dan kijk je of het geheel falsy is. Bovendien is het nu een "negated condition" en dat is niet lekker leesbaar ([uitleg](https://eslint.org/docs/2.0.0/rules/no-negated-condition))